### ISSUE TITLE
not very easy to use

### ISSUE THREAD

**Issue by playgithub** [adolfintel/Windows10-Privacy#40]
so many commands, and misc things
If there is a GUI application for this, it'll be perfect.

**Comment by adolfintel** [comment]
I know, I'll do it eventually

**Comment by lazerl0rd** [comment]
@playgithub I integrated most of this and a bit more into https://gitlab.com/ZKP/ScriptWINg, (available at lasterhub.me.

**Comment by adolfintel** [comment]
@lazerl0rd the link is broken

**Comment by lazerl0rd** [comment]
Oops, my mistake @adolfintel @playgithub. Link fixed.

**Comment by adolfintel** [comment]
@lazerl0rd Just for curiosity, why is it called anti_piracy?

**Comment by lazerl0rd** [comment]
> @lazerl0rd Just for curiosity, why is it called anti_piracy?

I really didn't think it through. It was originally anti_windowspiracykiller or something and then I shortened it down wrongly. It just stuck in the end since I have a complex system of post-OTA/Update scripts that run and decided was too much effort to rename it.

TL;DR I didn't read through before changing and pushing.
